src/integration-test/java/uk/gov/hmcts/juror/api/moj/controller/JurorManagementControllerITest.java

```diff
@@ -1788,6 +1788,7 @@ void confirmAttendanceHappy() {
             assertThat(appearance.getTimeOut()).isEqualTo(LocalTime.of(17, 00));
             assertThat(appearance.getAppearanceStage()).isEqualTo(EXPENSE_ENTERED);
             assertThat(appearance.getAttendanceAuditNumber()).isEqualTo("J00123456");
+            assertThat(appearance.getSatOnJury()).isTrue();
```

Comment:
can you update the other tests to make sure the property is not true (validate the negative scenarios)

Reply:
👍